code/src/routes/protectedRoutes.tsx

```diff
@@ -12,7 +12,10 @@ const protectedRoutes: RouteObject[] = [
   {
     path: routePaths.orders.path,
     element: (
-      <ProtectedRoute element={<OrdersPage />} allowedRoles={[ROLES.USER]} />
+      <ProtectedRoute
+        element={<OrdersPage />}
+        allowedRoles={[ROLES.USER, ROLES.ADMIN, ROLES.SHOP_MANAGER]}
```

Comment:
Can we add something like `ROLES.ALL`? We may want to add one some roles from scratch or add all roles and omit some role from all

Reply:
I will create new issue for it